client/src/Organisms/RegistrationForm/RegistrationFormSignup.js

```diff
@@ -15,7 +14,8 @@ const RegistrationFormSignup = ({
   initialState,
   editPhoneNumber,
 }) => {
-  const { handleInputChange, submitHandler, formData, errors } = useForm(
+  // useForm name is reserved for react-hook-form
```

Comment:
I think its not necessary to leave the comment

Reply:
Removed.